examples/wkwebview/wkwebview/LinkViewController.swift

```diff
@@ -10,42 +10,44 @@ import WebKit
 
 class LinkViewController: UIViewController, WKNavigationDelegate {
     
-    @IBOutlet var containerView : UIView? = nil
+    /// URLs matching this scheme will be intercepted & handled by this view controller
+    private static let linkScheme = "plaidlink"
     
     var webView: WKWebView!
     
     override func loadView() {
-        self.webView = WKWebView()
-        self.webView.navigationDelegate = self
-        self.view = webView
+        createWebview()
     }
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // load the link url
+        loadLinkModuleInWebview()
+    }
+    
+    private func createWebview() {
+        webView = WKWebView()
+        webView.navigationDelegate = self
+        webView.allowsBackForwardNavigationGestures = false
+        view = webView
+    }
+    
+    private func loadLinkModuleInWebview() {
         let linkUrl = generateLinkInitializationURL()
         let url = NSURL(string: linkUrl)
         let request = NSURLRequest(url:url! as URL)
-        self.webView.load(request as URLRequest)
-        self.webView.allowsBackForwardNavigationGestures = false
```

Comment:
I moved it to the `createWebview()` method -- can move it back here if you want, let me know.